lib/tests/changefeeds.rs

```diff
@@ -674,3 +675,87 @@ async fn changefeed_with_ts() -> Result<(), Error> {
 	assert_eq!(array.len(), 0);
 	Ok(())
 }
+
+#[tokio::test]
+async fn changefeed_with_original() -> Result<(), Error> {
+	if !FFLAGS.change_feed_live_queries.enabled() {
+		return Ok(());
+	}
+	let db = new_ds().await?;
+	let ses = Session::owner().with_ns("test").with_db("test");
+	// Enable change feeds
+	db.execute("DEFINE TABLE user CHANGEFEED 1h INCLUDE ORIGINAL;", &ses, None)
```

Comment:
We still say `INCLUDE ORIGINAL` even though we actually store diffs instead of the originals, just because changing the syntax is out of the scope of this PR (right?).

Reply:
Yes, exactly. Not great, but I can follow up. Or even change in this PR if we think it should be coupled. I don't think it should - it allows me to work independently of this change and PR lifecycle.